packages/edit-site/src/components/page-patterns/grid-item.js

```diff
@@ -52,6 +51,33 @@ import { store as editSiteStore } from '../../store';
 import { useLink } from '../routes/link';
 import { unlock } from '../../lock-unlock';
 
+/**
+ * Downloads a file.
+ * Also used in packages/list-reusable-blocks/src/utils/file.js.
+ *
+ * @param {string} fileName    File Name.
+ * @param {string} content     File Content.
+ * @param {string} contentType File mime type.
+ */
+function download( fileName, content, contentType ) {
+	const file = new window.Blob( [ content ], { type: contentType } );
+
+	// IE11 can't use the click to download technique
+	// we use a specific IE11 technique instead.
+	if ( window.navigator.msSaveOrOpenBlob ) {
```

Comment:
Given we don't officially support IE11 anymore I wonder if we can remove this IE specific code?

Reply:
Oh yeah, good call!

Reply:
Tested on Edge just in case. LGTM to remove

Reply:
Still tested well for me across chrome/safari/firefox/safari with the IE code removed.